product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -88,18 +88,21 @@ To delete a project that you created:
 1. Go to **Settings** on the left-side navigation.
 1. From the **Settings** list, select **Security**.
 1. Select **Add a key**. 
-1. On the **Add a key** page, select:
-   - Select the **Provider & Region**
-   - Select **Next**
-   - Enter AWS Key Management System ARN
-   - Enter a friendly name for your key
-   - Select **Finish**
+1. On the **Add a key** page, select the **Cloud Service Provider**. 
+1. Select the **Region** for the key. The interface only displays the regions available in the cloud account you configured.
+   !!! Note 
+   GCP offers the option to configure global keys. If you require a global key, do not select any region.
+   !!! 
+1. Complete the remaining fields according to your cloud provider. 
+1. Select **Add Key** to finalize the configuration.
 
-Now, use this TDE key to create a cluster. For more information, see [Creating a cluster](/biganimal/release/getting_started/creating_a_cluster/#security).
+Now, use this TDE key to [create a cluster](/biganimal/release/getting_started/creating_a_cluster/#security).
```

Comment:
Add a note above this: 

Note for GCP keys:
If the key you added was created in a different Google Cloud Platform account than the cluster, ensure you enable the [Cloud KMS API](https://cloud.google.com/kms/docs/create-encryption-keys#before-you-begin) in the Google console before assigning it to your cluster in BigAnimal.